tbtaf/common/test.py

```diff
@@ -4,16 +4,14 @@
 @author: S41nz
 '''
 
+from __future__ import absolute_import
 from abc import ABCMeta, abstractmethod
-from _pyio import __metaclass__
+import six
 
```

Comment:
Is this package included by default or should we document it in the README?

Reply:
It's already documented on README and requirements.txt